libs/utils/utils.js

```diff
@@ -998,6 +998,21 @@ export function scrollToHashedElement(hash) {
   });
 }
 
+function logPagePerf() {
+  const isChrome = () => {
+    const nav = window.navigator;
+    return nav.userAgent.includes('Chrome') && nav.vendor.includes('Google');
+  };
+
+  if (getMetadata('pageperf') === 'on' && isChrome()) {
```

Comment:
Minor nit: I'd guess the feature would be disabled more often than not, right? In which case maybe we can do an early return based on the `pageperf` metadata.
```suggestion
  if (getMetadata('pageperf') !== 'on') return;

  const isChrome = () => {
    const nav = window.navigator;
    return nav.userAgent.includes('Chrome') && nav.vendor.includes('Google');
  };

  if (!isChrome()) return;
```